src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -105,6 +109,20 @@ class SettingsMigration : Activity {
         }
   }
 
+  private fun migrateUrlsToCodebaseNames(project: Project) {
```

Comment:
Please add a test for that.
Especially considering those cases:
* previously added codebase name without http://
* previously added codebase name with upercase address

Reply:
done